general-concepts/use-flags/text.xml

```diff
@@ -194,6 +194,39 @@ To get a flag added to <c>use.mask</c>, ask the relevant arch team.
 </body>
 </section>
 
+<section>
+<title>IUSE defaults</title>
+<body>
+
+<p>
+Add <c>+</c> or <c>-</c> before the name of the use flag in <c>IUSE</c> to turn
+it on or off by default.
+</p>
+
+<important>
+The default USE-ordering is
+<c>USE_ORDER="env:pkg:conf:defaults:pkginternal:features:repo:env.d"</c>
+with the highest precedence listed first (see man make.conf).
```

Comment:
I'm not sure if this is explanatory enough. We should either import the whole text from make.conf and not mention it IMO.

Reply:
I have deleted the first `important` box, and moved the reference to make.conf(5) to the second one.